src/spinline.jquery.js

```diff
@@ -78,7 +78,8 @@
                                 position: 'absolute',
                                 left: 0,
                                 width: 0,
-                                boxShadow: "2px 0 20px " + settings.color
+                                boxShadow: "2px 0 20px " + settings.color,
+				'z-index': 2147483647
```

Comment:
This one is incorrect. Also, such big z-index is not good. If you need the spinner to be under some drop down elements?